src/api/app/services/workflows/yaml_to_workflows_service.rb

```diff
@@ -0,0 +1,24 @@
+module Workflows
+  class YAMLToWorkflowsService
+    def initialize(yaml_file:, scm_extractor_payload:)
+      @yaml_file = yaml_file
+      @scm_extractor_payload = scm_extractor_payload
+    end
+
+    def call
+      create_workflows
+    end
+
+    private
+
+    def create_workflows
```

Comment:
This isn't creating workflows. It is initializing workflows since it's doing `Workflow.new(...)`.